source/renderer/app/components/staking/redeem-itn-rewards/Step1ConfigurationDialog.scss

```diff
@@ -38,20 +38,6 @@
     margin-bottom: 17px;
   }
 
-  .walletsDropdown {
```

Comment:
This state was implemented locally - moving this functionality to `react-polymorph` so it can be reused.